server/sonar-server/src/main/java/org/sonar/server/permission/index/PermissionIndexer.java

```diff
@@ -91,16 +81,9 @@ public PermissionIndexer(DbClient dbClient, EsClient esClient, Collection<Author
 
   @Override
   public void indexOnStartup(Set<IndexType> emptyIndexTypes) {
-    Future<?> submit = executor.submit(() -> {
       List<Dto> authorizations = getAllAuthorizations();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Make this line start at column 5. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)